From 2276ca1f58ba9fd4701b2cf036d2b239fcb03f3b Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.org>
Date: Mon, 26 Nov 2018 21:01:18 +0000
Subject: [PATCH 096/185] Revert "build/arm64: Add rules for .dtbo files for
 dts overlays"

DT build rules are now in the common top-level Makefile.

This reverts commit dce5b0fbdd2174f7fc9b27f4e89877a33169239a.
---
 arch/arm64/Makefile | 14 --------------
 1 file changed, 14 deletions(-)

diff --git a/arch/arm64/Makefile b/arch/arm64/Makefile
index 64d9be3..b025304 100644
--- a/arch/arm64/Makefile
+++ b/arch/arm64/Makefile
@@ -145,20 +145,6 @@ Image.%: Image
 zinstall install:
 	$(Q)$(MAKE) $(build)=$(boot) $@
 
-%.dtb: scripts
-	$(Q)$(MAKE) $(build)=$(boot)/dts $(boot)/dts/$@
-
-%.dtbo: | scripts
-	$(Q)$(MAKE) $(build)=$(boot)/dts MACHINE=$(MACHINE) $(boot)/dts/$@
-
-PHONY += dtbs dtbs_install
-
-dtbs: prepare scripts
-	$(Q)$(MAKE) $(build)=$(boot)/dts
-
-dtbs_install:
-	$(Q)$(MAKE) $(dtbinst)=$(boot)/dts
-
 PHONY += vdso_install
 vdso_install:
 	$(Q)$(MAKE) $(build)=arch/arm64/kernel/vdso $@
-- 
1.9.1

